slap_plugin.go

```diff
@@ -30,13 +30,18 @@ func init() {
 		"destroys $USER",
 		"discombobulates $USER",
 		"does far worse, taking $USER's system and installing Windows",
+		"drinks $USER coffee",
+		"eats $USER pizza",
```

Comment:
These should be `$USER's` in order to be valid English. :P

Reply:
Now It's solved c: